index.d.ts

```diff
@@ -0,0 +1,17 @@
+import { ConstructorOptions } from "logdna"
+import Transport from "winston-transport";
+
+declare class LogDNATransport extends Transport {
+  constructor(options: LogDNATransport.TransportOptions);
+}
+
+declare namespace LogDNATransport {
+  interface TransportOptions extends Transport.TransportStreamOptions, ConstructorOptions {
+    /** The LogDNA API key. */
+    key: string;
+    /** The name of this transport. */
```

Comment:
Can you add the default values for these optional variables in the new line of the comment section?

For `name` it's `LogDNA`;
For `level` it's `debug`;
For `index_meta` it's `false`.

Reply:
I believe this should be added directly in the `TransportStreamOptions`

Reply:
@Sytten, I am so sorry for the late response but we can say that for `level` but not for others based on [this](https://github.com/winstonjs/winston-transport/blob/master/index.d.ts).

So, you need to add `index_meta` as well + the default values for these optional variables in the new line of the comment section. Thanks a lot in advance!

Reply:
I dont agree, those are defined in `ConstructorOptions`.